nc/data/importer.py

```diff
@@ -7,17 +7,64 @@
 from django.conf import settings
 from django.db import connections
 
+from tsdata.dataset_facts import compute_dataset_facts
 from tsdata.sql import drop_constraints_and_indexes
-from tsdata.utils import call, line_count, download_and_unzip_data
-
+from tsdata.utils import call, flush_memcached, line_count, download_and_unzip_data, unzip_data
+from nc.models import Agency, Search, Stop
+from nc.prime_cache import run as prime_cache_run
+from .download_from_nc import nc_download_and_unzip_data
 
 logger = logging.getLogger(__name__)
 
+MAGIC_NC_FTP_URL = 'ftp://nc.us/'
+
+
+def run(url, destination=None, zip_path=None, min_stop_id=None,
+        max_stop_id=None, prime_cache=True):
+    """
+    Download NC data, extract, convert to CSV, and load into PostgreSQL
+
+    :param url: if not None, zip will be downloaded from this URL; this can
+      either be a URL supported by the requests library OR the special URL
+      MAGIC_NC_FTP_URL, in which case the zip will be downloaded from the state
+      of North Carolina server.
+    :param destination: directory for unpacking zip and creating other
+      files; pass None to create a temporary file
+    :param zip_path: path to previously-downloaded zip
+    :param prime_cache: whether or not to prime the query cache for "big"
+      NC agencies after import
+    :param max_stop_id: only process stops with ids <= this value; this is to
+      save time for developers by reducing the amount of data to import
+    :param min_stop_id: only process stops with ids >= this value; this is to
+      save time for developers by reducing the amount of data to import
+    """
+    if not url and not destination:
+        raise ValueError('destination must be provided when download is False')
```

Comment:
fixed in https://github.com/OpenDataPolicingNC/Traffic-Stops/pull/200/commits/a81f242e333f434a38dcccdf5fe74aa830a32dd1

(yeah, an earlier version had a `download` parameter)